internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestHybrid.java

```diff
@@ -1716,6 +1717,8 @@ private static VeniceClusterWrapper setUpCluster(boolean enablePartitionWiseShar
           KafkaConsumerService.ConsumerAssignmentStrategy.PARTITION_WISE_SHARED_CONSUMER_ASSIGNMENT_STRATEGY.name());
     }
     cluster.addVeniceServer(new Properties(), serverProperties);
+    // Enable blob files in one server
+    serverProperties.setProperty(ROCKSDB_BLOB_FILES_ENABLED, "true");
```

Comment:
May be we could add a JMH benchmark test for blob-file mode.

Reply:
It is hard to test it properly as it is mainly about write amplification.
I would prefer to validate it in a perf test env.